Pratiksha/Bootstrap/Assignment5/Assignment5.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```

Comment:
check indentation head is a child tag of html tag make this change in every other assignments